### ISSUE TITLE
Remarkable 2

### ISSUE THREAD

**Issue by spraetor** [Evidlo/remarkable_keyboard#4]
I've tried this code on my remarkable 2 and it works after some minor changes:

- use the device `/dev/input/event2`
- touchscreen resolution is the same as the display resolution
- either mirror y axis, or rotate keyboard and mirror x axis

**Comment by flomlo** [comment]
Yeah, I've encountered similarish issues, nothing major. @Evidlo: Are you interested in a PR or shall I just fork?

We can have device autodetection with something like (copied from my project `rm2canvas`):
```
def check_model(rm_hostname, rm_user="root"):
    try:
        command = f"ssh -o ConnectTimeout=2 {rm_user}@{rm_hostname} cat /proc/device-tree/model"
        model = subprocess.run(command, shell=True, check=True, capture_output=True)
        return model.stdout[:14].decode("utf-8")
    except subprocess.CalledProcessError:
        print(f"Error: Can't connect to reMarkable tablet on hostname : {rm_hostname}")
        os._exit(1)
```

rest should be moderately trivial.

I can write the changes and make a PR, but I can't test it on rm1-devices.

**Comment by Evidlo** [comment]
Is this still an issue?  This should be handled at https://github.com/Evidlo/remarkable_mouse/blob/30b8a7b7131f652a7b162144a7b4c911fcc6eb7c/remarkable_mouse/remarkable_mouse.py#L109-L116
